Dockerfile

```diff
@@ -12,4 +12,11 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/ratelimit -ldflags="-w -s" -v g
 
 FROM alpine:3.11 AS final
 RUN apk --no-cache add ca-certificates
+
+FROM ubuntu:latest as install
```

Comment:
why do we need `as install` here

looks like this label is not used

Reply:
removed